core/tests/performance_tests/splash_test.py

```diff
@@ -0,0 +1,31 @@
+"""Tests for the splash page"""
+
+import unittest
+from test_utils import TestBase
+
+
+class SplashPageTest(TestBase):
+    """Tests for `splash page`."""
+
+    # how do I call setUp only once before all the test cases.
```

Comment:
@seanlip had a discussion with @BenHenning, we'll use setUp() and write separate tests for total size, load time and other metrics.